src/renderer/gdi/paint.cpp

```diff
@@ -543,20 +543,24 @@ bool GdiEngine::FontHasWesternScript(HDC hdc)
     };
     const auto DrawCurlyLine = [&](const auto x, const auto y, const auto cCurlyLines) {
         const auto curlyLineWidth = fontWidth;
-        const auto curlyLineHalfWidth = lrintf(curlyLineWidth / 2.0f);
-        const auto controlPointHeight = gsl::narrow_cast<long>(std::floor(3.5f * _lineMetrics.curlylinePeakHeight));
+        const auto curlyLineHalfWidth = std::lround(curlyLineWidth / 2.0f);
+        const auto controlPointHeight = std::lround(3.5f * _lineMetrics.curlylinePeakHeight);

```

Comment:
Do we have to ensure that these two are at least 1?

Reply:
(Well, probably only the former.)

Reply:
The fact that `curlyLineWidth == fontWidth == CellWidth`, these variables should always be greater than 1, no?

Do we expect cell width to be less than 2px?

Reply:
Oh, when I first read this, I thought that the width of the line is its "height", since you commonly call it "line width" or "stroke width", etc.